### PR TITLE
Name and age added in #container

### PR DESCRIPTION
**Pull request by agpranjal2000**
### The following changes have been made in index.html
+ Background color changed to **blue**
+ **Name / Age** added in #container

### CODE REVIEW

**Comment by agpranjal2000** [comment]
Yoo is anybody listening ??

**Comment by agpranjal** [comment]
> Yoo is anybody listening ??

###Yes.
I will merge the request soon.
Thank you.

**Review by agpranjal — CHANGES_REQUESTED** [review]
### Further changes required.
On a second thought, change the background color property to hex value

File: style.css [hunk 1]
```diff
@@ -1,5 +1,9 @@
+* {
+    color: white;
+}
+
 body {
-    background-color: #dddddd;
```
Comment (agpranjal) [inline style.css:2]: ### Nord color
Change this to #4C566A

File: style.css [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+    background-color: blue;
```
Comment (agpranjal) [inline style.css:6]: ### Nord color
Change this to #4C566A

**Review by agpranjal — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (agpranjal2000) [reply style.css:6]: OK.

**Review by agpranjal — APPROVED** [review]
### Perfect

**Pull request merged**